.github/workflows/windows-10-qt6.yml

```diff
@@ -0,0 +1,156 @@
+name: Windows 10 (64-bit, Qt6)
+
+on:
+  push:
+    branches: master
+    tags: 'v*.*.*'
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - 'LICENSE'
+      - '.gitignore'
+  pull_request:
+    branches: master
+    paths-ignore:
+      - '**.md'
+      - '**.txt'
+      - 'LICENSE'
+      - '.gitignore'
+
+defaults:
+  run:
+    shell: bash
+
+env:
+  BUILD_TARGET: windows-10-x86_64-qt6
```

Comment:
Is this too unwieldy of a release package name? I was hoping to avoid any confusion about what zip to download by making this as specific as possible.